crates/bevy_input/src/gamepad.rs

```diff
@@ -243,6 +261,14 @@ impl ButtonAxisSettings {
     }
 }
 
+fn scale(value: f32, low: f32, high: f32) -> f32 {
```

Comment:
What was the thinking behind including these functions at the module level? My instinct would be to make them private functions on `AxisSettings`, but I'm not super experienced with Rust conventions.

Reply:
Not experienced as well :)
As I was needing those in both `AxisSettings` and `ButtonAxisSettings` I include those functions at the module level, but kept them private.
But if it's better to duplicate them in both `AxisSettings` and `ButtonAxisSettings`, I'll do it!

Reply:
Ah, I didn't realize they would be used for both. In that case, I think this is fine.